eth/beacon/helpers.py

```diff
@@ -402,6 +404,32 @@ def get_domain(fork_data: 'ForkData',
         slot,
     ) * 4294967296 + domain_type
 
+def verify_slashable_vote_data(state: BeaconState, vote_data: SlashableVoteData) -> bool:
+    vote_count = len(vote_data.aggregate_signature_poc_0_indices) + len(vote_data.aggregate_signature_poc_1_indices)
+    if vote_count > MAX_CASPER_VOTES:
+        return False
+
+    pubkeys = [
+        bls.aggregate_pubkeys([state.validators[i].pubkey for i in vote_data.aggregate_signature_poc_0_indices]),
+        bls.aggregate_pubkeys([state.validators[i].pubkey for i in vote_data.aggregate_signature_poc_1_indices])
+    ]
```

Comment:
In Py-EVM we like `tuple` more than `list` for immutability. 

Refactoring idea 1 - Formatting
```python
pubkeys = tuple(
    tuple(
        state.validators[i].pubkey
        for i in vote_data.aggregate_signature_poc_0_indices
    ),
    tuple(
        state.validators[i].pubkey
        for i in vote_data.aggregate_signature_poc_1_indices
    ),
)
```

Refactoring idea 2 - Adding new helper function
```python
@to_tuple
def get_pubkey_from_indices(validators: Sequence[ValidatorRecord], indices: Sequence[int]) -> Iterable[int]:
    for index in vote_data.aggregate_signature_poc_0_indices]:
        yield validators[index]
```

Reply:
i've seen this `@to_tuple` decorator... i'll look into this method